docs/reference/setup/install/docker.asciidoc

```diff
@@ -283,7 +283,7 @@ method, you can also configure this by using the `ES_JAVA_OPTS` environment
 variable to set the heap size. For example, to use 16 GB, specify
 `-e ES_JAVA_OPTS="-Xms16g -Xmx16g"` with `docker run`. Note that while the
 default root `jvm.options` file sets a default heap of 1 GB, any value you set
-in `ES_JAVA_OPTS` will override it.
+in `ES_JAVA_OPTS` will override it. The yaml file above (docker-compose.yml) sets the size to 512 megabytes.
```

Comment:
Since the preceding text refers to "GB", lets use "MB" here. Let's also uppercase "YAML".

```suggestion
in `ES_JAVA_OPTS` will override it. The `docker-compose.yml` file above sets the heap size to 512 MB.
```